libs/net/dal/Services/ReportInstanceService.cs

```diff
@@ -127,6 +127,8 @@ public override ReportInstance Update(ReportInstance entity)
         });
         entity.ContentManyToMany.ForEach(ric =>
         {
+            // Duplicate content can be in multiple sections, so we grab the first copy.
```

Comment:
Fixing issue where saving a report fails if the edited content is in more than one section.